src/containers/Application/ApplicationWrapper.tsx

```diff
@@ -42,6 +44,9 @@ const ApplicationWrapper: React.FC = () => {
       <Route exact path={`${path}/summary/submission`}>
         <ApplicationSubmissionNEW structure={structure} />
       </Route>
+      <Route path={`${path}/review`}>
+        <ReviewWrapper />
```

Comment:
Would need structure passed through

Reply:
Oh! I was thinking we would rebuild the structure on the ReviewWrapper. Wasn't that the idea?

Reply:
The idea was for the BaseStructure to still come from ApplicationWrapper, it will get populated with other bits by useGetReviewStructure

Reply:
Ah, yea. I see. Will do this change in the evening.

Reply:
Done: 37155ad